onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1615,6 +1618,16 @@ static void LogDeprecationWarning(
 #endif
 
 void addGlobalMethods(py::module& m) {
+  m.def("set_global_thread_pool_sizes", [](int intra_op_num_threads, int inter_op_num_threads) {
+          static std::mutex global_thread_pool_mutex;
```

Comment:
It's safe to define such things as global vars. But, what does this mutex protect for?